### DISCUSSION TITLE
Font Sizing

### DISCUSSION THREAD

**Discussion by MikeGriffis** [bdtc/lwarp#29] [Q&A]
Can the tutorial be tweaked, enhanced to show how to control font sizes?  A few things look out of size in HTML.  When I follow the tutorial (math as SVG images), the inline math looks right (consistent with the paragraph) but the equation math looks much smaller.  The equation number is smaller too.  When I am viewing this page, the contents (left margin) is small.  I've looked at the documentation (secs 8.7.4, 84.3, 84.4, 96) but not really getting it.  I viewed it from different computers with same result.

**Comment by bdtc** [comment]
If the browser's display font is large then that can make the SVG equations look small.  Try a browser font size around 12 point, for example.

`\LateximageFontSizeName` and `\LateximageFontScale` adjust the sizes when generating SVG.

Equations can scale if the window is too narrow.

**Reply by MikeGriffis** [reply]
After some experimenting, here is what I've found.

- it is true if I decrease the browser font, then the SVG equations become the same size as the inline text.  But everything is small.  All webpages are small.  I have to put it back to 16 in order to read any webpage.
- I tried renew on the two variables.  The `\LateximageFontSizeName` did not seem to do anything.  The `\LateximageFontScale` did change the inline math font size but not the equation font size (consistent with your documentation).
- While compiling, it suggested that I include 'graphics' or 'graphicx' which I did.  Did not help.
- I tried recompiling with mathjax enabled.  Now, equation font size (with eqn number) is same size as inline and paragraph text.  This seems to work.

In summary, it seems difficult to scale equation fonts (and equation numbers) for the SVG fonts.

**Reply by bdtc** [reply]
There is a trend in website design to try to adapt to the browser window size.  Lwarp scales display equations to fit into the width of the browser window, if possible.  Imagine if you had to scroll horizontally to see the equation number, for example.
